src/styles/partials/components/_breadcrumbs.scss

```diff
@@ -1,3 +1,5 @@
+$breadcrumb-arrow-size: 20px;
+
```

Comment:
Should this be moved into a variables page? I feel like if its used enough to warrant a variable then maybe it needs to be available globally. But not blocking.

Reply:
not sure, its not a bad idea, but I think this is just specific to breadcrumbs right now, and doesn't match up with the default. Worth coming back to.